app/src/main/res/values/arrays.xml

```diff
@@ -15,4 +15,14 @@
         <item>battery_power</item>
     </string-array>
 
+    <string-array name="theme_names">
+        <item>Light</item>
+        <item>Dark</item>
+    </string-array>
+
+    <string-array name="theme_values">
+        <item>1</item>
+        <item>2</item>
+    </string-array>
```

Comment:
Why the numeric values? Can't we just use text? Or is this for the purpose of translation vs value in storage?

If yes, I don't see the preference names being in translation or is that automatic?

Reply:
The reason why these are numbers is so that they can be used with android's DayNight theme. They're based on the [constant values on AppCompatDelegate](https://developer.android.com/reference/android/support/v7/app/AppCompatDelegate.html#mode_night_auto).

Reply:
I should note that they don't show up as numbers in the UI. They're numbers so that this value can be passed to `AppCompatDelegate.setDefaultNightMode` without having to convert it first.

Reply:
Question about translations still stands

Reply:
You're right. I've moved the strings to strings.xml